playbooks/collect_instance_statistics/query_elasticsearch.py

```diff
@@ -0,0 +1,91 @@
+#!/edx/bin/python.edxapp
```

Comment:
@pcockwell, there will be no `/edx/bin/python.edxapp` on the logs server. So this has to be updated.

Reply:
I've yet to be able to access the server. Do you know what python executable will be available?

Reply:
@pcockwell, I will set up access for the logs.opencraft.com server today. The server runs Ubuntu 18.04 which has Python 3.6.x.